src/steps/MatchColumnsStep/utils/setColumn.ts

```diff
@@ -1,19 +1,30 @@
 import type { Field } from "../../../types"
-import { Column, ColumnType, MatchColumnsProps } from "../MatchColumnsStep"
+import { Column, ColumnType, MatchColumnsProps, MatchedOptions } from "../MatchColumnsStep"
 import { uniqueEntries } from "./uniqueEntries"
 
 export const setColumn = <T extends string>(
   oldColumn: Column<T>,
   field?: Field<T>,
   data?: MatchColumnsProps<T>["data"],
+  autoMapSelectValues?: boolean,
 ): Column<T> => {
   switch (field?.fieldType.type) {
     case "select":
+      const options = field.fieldType.options
+      const uniqueData = uniqueEntries(data || [], oldColumn.index) as MatchedOptions<T>[]
+      const matchedOptions = autoMapSelectValues
+        ? uniqueData.map((option) => {
+            const value = options.find((o) => o.value == option.value || o.label == option.entry)?.value
```

Comment:
double equals

Reply:
matching values is good, I'm not sure about matching label to entry?

Reply:
Thanks, fixed!